subprojects/docs/docs.gradle

```diff
@@ -42,6 +38,7 @@ apply plugin: 'base'
 apply plugin: 'pegdown'
 apply plugin: 'jsoup'
 apply plugin: 'javascript-base'
+apply plugin: 'org.asciidoctor.convert'
```

Comment:
Do we still need to apply the plugin or do we only need the types? We should get those from the buildSrc dependencies.

Reply:
Yes it does some setup. Creating the extension, a configuration and some dependency management. I would leave the `apply` in to use as much of the original plugin as possible.